web/src/components/ThemeContainer.tsx

```diff
@@ -35,6 +35,38 @@ const createTheme = (
       light: buildConfig().lightTheme,
       dark: buildConfig().darkTheme,
     },
+    components: {
+      MuiTextField: {
+        styleOverrides: {
+          root: {
+            '& .MuiOutlinedInput-input': {
+              '&:focus': {
+                outline: 'none',
```

Comment:
🔧 We manually add the outlines in the `Layout.tsx`, I would strongly prefer if we remove them there instead of here :)